### ISSUE TITLE
Example.com domain

### ISSUE THREAD

**Issue by florianroesler** [andreis/disposable-email-domains#20]
Hey,
I just used the list and saw that example.com is on the list, which as far as I know it is not a disposable email domain but instead a reserved domain by IANA for testing etc.

I think it would make sense to remove it :)

**Comment by smeinecke** [comment]
Thanks for your inquiry! You are right, the domains example.com/.net/.org are "blocked" by ICANN/IANA for testing purposes and do not allow the reception of e-mails. 

Since most users use this list to check whether an "invalid" mail host has been specified by a customer, these hosts should also be included in the list, even if it is not really a "disposable" mail host. Sorry :(

**Comment by florianroesler** [comment]
I was not aware of this. I assumed that all of the domains are by disposable email providers. In my understanding, using this list to identify 'invalid' email hosts is not really a viable use case because you have an infinite amount of those out there (any combination of characters with a valid tld for example). I might be completely off track here but whether something is a valid email host can only be evaluated with a confirmation mail or at least some DNS checks. Following this logic, using test@example1234.com would be a valid email host, even though it might not be able to receive emails.

What I wrote above is not meant as criticism to this project, I just wanted to get some clarification ;)
I completely respect your decision. For my own use case, I will just remove the respective lines in my local environment.

Just FYI:
I came across this issue because we use the list for email validation of our signups. In our testing environment, we use a Ruby gem called Faker and when generating 'safe' fake email addresses they use the 'example.com' domain (https://github.com/faker-ruby/faker/blob/master/doc/default/internet.md). So in our test environment, all user creations became invalid because of that :)
